src/layouts/main.html

```diff
@@ -28,7 +28,7 @@
   </if>
   <block name="head"></block>
 </head>
-<body class="m-0 p-0 w-full break-word antialiased {{ page.bodyClass || '' }}">
+<body class="m-0 p-0 w-full [word-break:break-word] [-webkit-font-smoothing:antialiased] {{ page.bodyClass || '' }}">
```

Comment:
Yes, we keep it as a basic example of doing imports and adding custom utilities. Just couldn't think of another property that we might need in email and isn't already in Tailwind...

Reply:
Ok, that's what I expected. I just wanted to make sure since it's not being used anywhere, and I saw you already removed the `resets.css` file, which could maybe also stay as an example.